Dockerfile

```diff
@@ -25,23 +25,17 @@ RUN apt-get update && \
 # Installing freesurfer
 RUN curl -sSL https://surfer.nmr.mgh.harvard.edu/pub/dist/freesurfer/6.0.1/freesurfer-Linux-centos6_x86_64-stable-pub-v6.0.1.tar.gz | tar zxv --no-same-owner -C /opt \
     --exclude='freesurfer/average' \
+    --exclude='freesurfer/data' \
     --exclude='freesurfer/diffusion' \
     --exclude='freesurfer/docs' \
+    --exclude='freesurfer/fsafd' \
     --exclude='freesurfer/fsfast' \
     --exclude='freesurfer/lib/cuda' \
     --exclude='freesurfer/lib/qt' \
     --exclude='freesurfer/matlab' \
-    --exclude='freesurfer/mni/share/man' \
-    --exclude='freesurfer/subjects/fsaverage_sym' \
-    --exclude='freesurfer/subjects/fsaverage3' \
-    --exclude='freesurfer/subjects/fsaverage4' \
-    --exclude='freesurfer/subjects/cvs_avg35' \
-    --exclude='freesurfer/subjects/cvs_avg35_inMNI152' \
-    --exclude='freesurfer/subjects/bert' \
-    --exclude='freesurfer/subjects/lh.EC_average' \
-    --exclude='freesurfer/subjects/rh.EC_average' \
-    --exclude='freesurfer/subjects/sample-*.mgz' \
-    --exclude='freesurfer/subjects/V1_average' \
+    --exclude='freesurfer/mni' \
+    --exclude='freesurfer/subjects' \
```

Comment:
you can't remove `freesurfer/subjects` in full, `fsaverage{5,6,}` have to be there

Reply:
For niworkflows?

Reply:
In principle not, maybe you can get away with `mkdir -p /opt/freesurfer/subjects`.

Reply:
Oh, I see. The failing test.